frame/vesting/src/tests.rs

```diff
@@ -1155,3 +1155,39 @@ fn vested_transfer_less_than_existential_deposit_fails() {
 		);
 	});
 }
+
+#[test]
+fn remove_vesting_schedule() {
+	ExtBuilder::default().existential_deposit(ED).build().execute_with(|| {
+		let user3_free_balance = Balances::free_balance(&3);
+		let user4_free_balance = Balances::free_balance(&4);
+		assert_eq!(user3_free_balance, 256 * 30);
+		assert_eq!(user4_free_balance, 256 * 40);
+		// Account 4 should not have any vesting yet.
+		assert_eq!(Vesting::vesting(&4), None);
+		// Make the schedule for the new transfer.
+		let new_vesting_schedule = VestingInfo::new(
+			256 * 5,
+			64, // Vesting over 20 blocks
+			10,
+		);
+		assert_ok!(Vesting::vested_transfer(Some(3).into(), 4, new_vesting_schedule));
+		// Now account 4 should have vesting.
+		assert_eq!(Vesting::vesting(&4).unwrap(), vec![new_vesting_schedule]);
+		// Account 4 has 5 * 256 locked.
+		assert_eq!(Vesting::vesting_balance(&4), Some(256 * 5));
+		// Verify only root can call
+		assert_noop!(Vesting::force_remove_vesting_schedule(Some(4).into(), 4, 0), BadOrigin);
+		// Verifies that vesting schedule is removed
+		assert_ok!(Vesting::force_remove_vesting_schedule(RawOrigin::Root.into(), 4, 0));
```

Comment:
This should emit an event, or?  
You can get it with `System::events().last().unwrap()` and then assert that its the expected one.

Reply:
I was looking around other pallet and it looks to me that this is better way of doing this check:
```rust
System::assert_last_event(
	Event::VestingCompleted {
		account: 4
	}
	.into(),
);
```
If you agree, I will implement it this way.

Reply:
Yes thats even better :smile:

Reply:
This has been added in [7479a1](https://github.com/paritytech/substrate/pull/11910/commits/7479a1212ba53cb5c8e17003f59a0d1835a1bbcf) commit.